### PR TITLE
Fix flacky test

### PR DESCRIPTION
**Pull request by tivv**
Test was calling start twice (one start and one startAndWait)

### CODE REVIEW

File: kafka-plugins-0.10/src/test/java/io/cdap/plugin/KafkaStreamingSourceTest.java [hunk 1]
```diff
@@ -264,7 +264,7 @@ public void testKafkaStreamingSourceFromLastOffset() throws Exception {
     sparkManager.waitForStatus(true, 10, 1);
 
     // for this test we have to wait for pipeline to be in running state before sending data
-    sparkManager.startAndWaitForRun(ProgramRunStatus.RUNNING, 20, TimeUnit.SECONDS);
+    sparkManager.waitForRun(ProgramRunStatus.RUNNING, 20, TimeUnit.SECONDS);
```
Comment (rmstar) [inline kafka-plugins-0.10/src/test/java/io/cdap/plugin/KafkaStreamingSourceTest.java:267]: Were we starting the pipeline twice before your fix (first started in line 263)?

**Review by rmstar — COMMENTED** [review]
I see this failure on unit test run for this PR. Is that expected?
```
Tests run: 4, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 197.725 sec <<< FAILURE!
testKafkaStreamingSourceFromLastOffset(io.cdap.plugin.KafkaStreamingSourceTest)  Time elapsed: 124.705 sec  <<< ERROR!
java.util.concurrent.TimeoutException: Timeout occurred. Expected {1=samuel jackson, 2=dwayne johnson, 3=christopher walken} but found {}.
	at io.cdap.cdap.common.utils.Tasks.waitFor(Tasks.java:71)
	at io.cdap.cdap.common.utils.Tasks.waitFor(Tasks.java:92)
	at io.cdap.cdap.common.utils.Tasks.waitFor(Tasks.java:131)
	at io.cdap.plugin.KafkaStreamingSourceTest.testKafkaStreamingSourceFromLastOffset(KafkaStreamingSourceTest.java:271)
```

File: kafka-plugins-0.10/src/test/java/io/cdap/plugin/KafkaStreamingSourceTest.java (hunk 1 above)
Comment (tivv) [reply kafka-plugins-0.10/src/test/java/io/cdap/plugin/KafkaStreamingSourceTest.java:267]: We do `sparkManager.start()` in line 182 and `sparkManager.startAndWaitForRun` in here. startAndWaitForRun explicitly says it should not be called with another run in progress

Comment (tivv) [reply kafka-plugins-0.10/src/test/java/io/cdap/plugin/KafkaStreamingSourceTest.java:267]: Sorry, not 182, line  263

**Review by rmstar — APPROVED** [review]

**Comment by tivv** [comment]
I see Caused by: java.lang.IncompatibleClassChangeError: class io.cdap.cdap.app.runtime.spark.DelegatingSparkListener has interface org.apache.spark.scheduler.SparkListener as super class
I believe it's same problem with latest CDAPIO not available at sonatype

**Pull request merged**